com/edocti/jintro/Exercise6.java

```diff
@@ -0,0 +1,211 @@
+import java.lang.Math;
+
+public class Exercise6 {
+
+	// getName
+	public static String getName(String student) {
+		String[] lines = student.split("\n");
+		if (lines == null || lines.length == 0) {
+			return "You must insert the students !";
+		}
+		String names = student.split(",")[0].trim();
+		return names;
+	}
+
+	// getSurname
+	public static String getSurname(String student) {
+		String[] lines = student.split("\n");
+		if (lines == null || lines.length == 0) {
+			return "You must insert the students !";
+		}
+		String surnames = student.split(",")[1].trim();
+		return surnames;
+	}
+
+	// checkNotaLucrare1
+	public static boolean checkNotaLucrare1(String student) {
+		String notalucrare1 = (student.split(",")[2].trim());
+		if (notalucrare1 != null) {
+			return true;
+		} else
+			return false;
+	}
+
+	// checkNotaLucrare2
+	public static boolean checkNotaLucrare2(String student) {
+		String notalucrare2 = (student.split(",")[3].trim());
+		if (notalucrare2 != null) {
+			return true;
+		} else
+			return false;
+	}
+
+	// checkNotaTeza
+	public static boolean checkNotaTeza(String student) {
+		String notateza = (student.split(",")[4].trim());
+		if (notateza != null) {
+			return true;
+		} else
+			return false;
+	}
+
+	// getNotaLucrare1
+	public static int getNotaLucrare1(String student) {
+		String[] lines = student.split("\n");
+		if (lines == null || lines.length == 0) {
+			return -1;
+		}
+		int notalucrare1 = Integer.parseInt(student.split(",")[2].trim());
+		return notalucrare1;
+	}
+
+	// getNotaLucrare2
+	public static int getNotaLucrare2(String student) {
+		String[] lines = student.split("\n");
+		if (lines == null || lines.length == 0) {
+			return -1;
+		}
+		int notalucrare2 = Integer.parseInt(student.split(",")[3].trim());
+		return notalucrare2;
+	}
+
+	// getNotaTeza
+	public static int getNotaTeza(String student) {
+		String[] lines = student.split("\n");
+		if (lines == null || lines.length == 0) {
+			return -1;
+		}
+		int notateza = Integer.parseInt(student.split(",")[4].trim());
+		return notateza;
+	}
+
+	// getNumarAbsente
+	public static int getNumarAbsente(String student) {
+		int numarabsente = Integer.parseInt(student.split(",")[5].trim());
+		return numarabsente;
+	}
+
+	// getAverageNote
+	public static int[] getAverageNote(String student) {

```

Comment:
the name `student` is misleading; it gives me the impression that it's a single student.